installer/resources/pacbot_app/files/DB.sql

```diff
@@ -3354,5 +3354,8 @@ UPDATE cf_Target SET displayName='PostgreSQL' WHERE targetName in ('cloudsql_pos
 INSERT IGNORE INTO `cf_Target` (`targetName`, `targetDesc`, `category`, `dataSourceName`, `targetConfig`, `status`, `userId`, `endpoint`, `createdDate`, `modifiedDate`, `domain`,displayName)
 VALUES('workstation','Workstation','Compute','crowdstrike','{\"key\":\"_resourceid,cid\",\"id\":\"_resourceid\",\"name\":\"_resourcename\"}','enabled','admin@paladincloud.io',
 concat(@eshost,':',@esport,'/crowdstrike_workstation'),now(),null,'Infra & Platforms','Workstation');
+INSERT IGNORE INTO cf_Target (`targetName`,`targetDesc`,`displayName`,`category`,`dataSourceName`,`targetConfig`,`status`,`userId`,`endpoint`,`createdDate`,`modifiedDate`,`domain`)
```

Comment:
I thought we are not introducing a new "server" target type, instead are mapping CS "server" types to an appropriate aws, azure or gcp "server" target type?

Reply:
we are adding a server target type for cs as a finding type, so that the metadata are shipped and can be queried for other purposes(policy evaluations, etc). This follows the same model similar to other finding asset types(eg. azure_securitypricings). I'd suggest we can follow an iterative approach and address this when we do the delta engine 1.0